geomstats/_backend/autograd/__init__.py

```diff
@@ -384,12 +385,26 @@ def array_from_sparse(indices, data, target_shape):
     return array(coo_matrix((data, list(zip(*indices))), target_shape).todense())
 
 
+def tril_to_vec(x, k=0):
+    """ """
+    n = x.shape[-1]
+    rows, cols = tril_indices(n, k=k)
+    return x[..., rows, cols]
+
+
 def triu_to_vec(x, k=0):
+    """ """
     n = x.shape[-1]
     rows, cols = triu_indices(n, k=k)
     return x[..., rows, cols]
 
 
+def vec_to_diag(vec):
```

Comment:
How is this different from https://numpy.org/doc/stable/reference/generated/numpy.diag.html ? is it working for minbatches?